infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryService.java

```diff
@@ -319,30 +320,34 @@ private void connectToClusterManager() {
      * - Start Log Replication Server (receiver component)
      */
     private void bootstrapLogReplicationService() {
-        // Through LogReplicationConfigAdapter retrieve system-specific configurations (including streams to replicate)
-        LogReplicationConfig logReplicationConfig = getLogReplicationConfiguration(getCorfuRuntime());
+        try {
+            // Through LogReplicationConfigAdapter retrieve system-specific configurations (including streams to replicate)
```

Comment:
Do we also retrieve the version info through the adapter? If so, let's modify the comment a little bit

Reply:
Done